src/test/e2e/52_oci_compose_differential_test.go

```diff
@@ -14,102 +14,103 @@ import (
 	suite.NoError(err, "unable to extract zarf.yaml from the differential git package")
 	var normalZarfConfig types.ZarfPackage
-	err = utils.ReadYaml(filepath.Join(tmpPath, "zarf.yaml"), &normalZarfConfig)
+	err = utils.ReadYaml(filepath.Join(suite.tmpdir, "zarf.yaml"), &normalZarfConfig)
 	suite.NoError(err, "unable to read zarf.yaml from the differential git package")
-	os.Remove(filepath.Join(tmpPath, "zarf.yaml"))
+	os.Remove(filepath.Join(suite.tmpdir, "zarf.yaml"))
 
-	stdOut, stdErr, err = e2e.ExecZarfCommand("package", "create", anotherPackagePath, "--differential", "oci://"+suite.Reference.String()+"/podinfo-with-oci-flux:v0.24.0-amd64", "--insecure", "--set=PACKAGE_VERSION=v0.25.0", createOutFlag, "--confirm")
+	stdOut, stdErr, err = e2e.Zarf("package", "create", anotherPackagePath, "--differential", "oci://"+suite.Reference.String()+"/podinfo-with-oci-flux:v0.24.0-amd64", "--insecure", "--set=PACKAGE_VERSION=v0.25.0", "-o", suite.tmpdir, "--confirm")
 	suite.NoError(err, stdOut, stdErr)
 
 	// Extract and load the zarf.yaml config for the differentially built package
-	err = archiver.Extract(filepath.Join(tmpPath, differentialPackageName), "zarf.yaml", tmpPath)
+	err = archiver.Extract(filepath.Join(suite.tmpdir, differentialPackageName), "zarf.yaml", suite.tmpdir)
 	suite.NoError(err, "unable to extract zarf.yaml from the differential git package")
 	var differentialZarfConfig types.ZarfPackage
-	err = utils.ReadYaml(filepath.Join(tmpPath, "zarf.yaml"), &differentialZarfConfig)
+	err = utils.ReadYaml(filepath.Join(suite.tmpdir, "zarf.yaml"), &differentialZarfConfig)
 	suite.NoError(err, "unable to read zarf.yaml from the differential git package")
 
 	/* Perform a bunch of asserts around the non-differential package */
 	// Check the metadata and build data for the normal package
 	suite.Equal(normalZarfConfig.Metadata.Version, "v0.24.0")
 	suite.False(normalZarfConfig.Build.Differential)
 	suite.Len(normalZarfConfig.Build.OCIImportedComponents, 1)
-	suite.Equal(normalZarfConfig.Build.OCIImportedComponents["oci://127.0.0.1:555/helm-oci-chart:0.0.1-skeleton"], "helm-oci-chart")
+	suite.Equal(normalZarfConfig.Build.OCIImportedComponents["oci://127.0.0.1:555/helm-charts:0.0.1-skeleton"], "demo-helm-oci-chart")
 
 	// Check the component data for the normal package
 	suite.Len(normalZarfConfig.Components, 3)
-	suite.Equal(normalZarfConfig.Components[0].Name, "helm-oci-chart")
+	suite.Equal(normalZarfConfig.Components[0].Name, "demo-helm-oci-chart")
 	suite.Equal(normalZarfConfig.Components[0].Charts[0].URL, "oci://ghcr.io/stefanprodan/charts/podinfo")
-	suite.Equal(normalZarfConfig.Components[0].Images[0], "ghcr.io/stefanprodan/podinfo:6.3.3")
+	suite.Equal(normalZarfConfig.Components[0].Images[0], "ghcr.io/stefanprodan/podinfo:6.3.5")
 	suite.Len(normalZarfConfig.Components[1].Images, 2)
-	suite.Len(normalZarfConfig.Components[1].Repos, 4)
+	suite.Len(normalZarfConfig.Components[1].Repos, 2)
```

Comment:
You'll have to increment these checks when we add that branch ref'd repo to the test package.